src/main/java/com/google/graphgeckos/dashboard/fetchers/github/CommitData.java

```diff
@@ -0,0 +1,87 @@
+package com.google.graphgeckos.dashboard.fetchers.github;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CommitData {
+
+  // Name of the branch
```

Comment:
```suggestion
  // Name of the branch.
```

Reply:
I think, this is not required because this is not a complete sentence